auth-service/src/main/java/com/auth/authservice/security/JwtTokenUtil.java

```diff
@@ -72,20 +77,16 @@ public String generateToken(User user) {
         Map<String, Object> claims = new HashMap<>();
         claims.put(CLAIM_KEY_USERNAME, user.getUsername());
         claims.put(CLAIM_KEY_CREATED, new Date());
+        ArrayList<String> roles = new ArrayList<>();
```

Comment:
You can directly convert the roles to an array if you want Or use a stream to improve the code maybe

Reply:
something like this :

claims.put(CLAIM_KEY_ROLES, user.getRoles().stream().map(role -> role.getName()).toArray(String[]::new));